rke/k8s_service_options.go

```diff
@@ -380,6 +388,15 @@ func getKubeAPIOptions120() map[string]string {
 	return kubeAPIOptions
 }
 
+func getKubeAPIOptions121() map[string]string {
+	kubeAPIOptions := getKubeAPIOptions116()
+	// need to turn off this feature in k8s 1.20 since we are not using it https://github.com/kubernetes/kubernetes/pull/91921
```

Comment:
`kubeAPIOptions["feature-gates"] = "ServiceAccountIssuerDiscovery=false"`
Do we need this? I noticed that this is not the exact option compared to 120. Also comment needs to be updated to say 1.21 :)

Reply:
So ServiceAccountIssuerDiscovery went GA with 1.21 and is now true by default, we no longer need it.
Oh that's right, the comment is no longer required - removing